Release 236.0.0 (#4870)
MetaMask/core: packages/assets-controllers/CHANGELOG.md

## [Unreleased]

## [41.0.0]

### Changed

- **BREAKING**: The polling input accepted by `CurrencyRateController` is now an object with a `nativeCurrencies` property that is defined as a `string` array type ([#4852](https://github.com/MetaMask/core/pull/4852))
  - The `input` parameters of the controller's `_executePoll`, `_startPolling`, `onPollingComplete` methods now only accept this new polling input type.
  - The `nativeCurrency` property (`string` type) has been removed.
- **BREAKING**: `RatesController` now types the `conversionRate` and `usdConversionRate` in its state as `number` instead of `string`, to match what it was actually storing. ([#4852](https://github.com/MetaMask/core/pull/4852))
- Bump `@metamask/base-controller` from `^7.0.1` to `^7.0.2` ([#4862](https://github.com/MetaMask/core/pull/4862))
- Bump `@metamask/controller-utils` from `^11.4.0` to `^11.4.1` ([#4862](https://github.com/MetaMask/core/pull/4862))

> - Bump `@metamask/approval-controller` from `^7.1.0` to `^7.1.1` ([#4862](https://github.com/MetaMask/core/pull/4862))

Reviewer: ```suggestion
- Bump dev dependency `@metamask/approval-controller` from `^7.1.0` to `^7.1.1` ([#4862](https://github.com/MetaMask/core/pull/4862))
```

## [Unreleased]

## [41.0.0]

> ### Added
> 
> - `TokenListController` now exports the type for `TokensChainsCache` ([#4821](https://github.com/MetaMask/core/pull/4821))
> 

Reviewer: The `TokensChainsCache` type wasn't added as a package-level export, and therefore is unavailable to downstream consumers.

```suggestion
```

### Changed

> - **BREAKING**: `CurrencyRateController` now accepts an array of native currencies as its polling input. ([#4852](https://github.com/MetaMask/core/pull/4852))

Reviewer: This entry might need to provide some additional information on how it affects downstream consumers:

```suggestion
- **BREAKING**: The polling input accepted by `CurrencyRateController` is now an object with a `nativeCurrencies` property that is defined as a `string` array type ([#4852](https://github.com/MetaMask/core/pull/4852))
  - The `input` parameters of the controller's `_executePoll`, `_startPolling`, `onPollingComplete` methods now only accept this new polling input type.
  - `nativeCurrencies` replaces the `string`-type `nativeCurrency` property. 
```
